src/test/java/ReaderWriterTest.java

```diff
@@ -0,0 +1,126 @@
+package test;
+
+import static org.junit.Assert.assertEquals;
+
+import IO.DbReader;
+import IO.DbWriter;
+import data.Database;
+import data.DatabaseQuery;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+/* Validate that a .txt file can be properly be stored written and read from database.  */
```

Comment:
Could you check all your comments format?
/* comment. /*
one blank between /* and comment and */